client/src/constants.js

```diff
@@ -136,6 +137,15 @@ export const appRoutes = [
     icon: MdGavel,
     active: false,
   },
+  {
+    name: 'Planning',
+    protected: true,
+    allowedRoles: rolesAvailable,
```

Comment:
I believe only Finance and Admin should have access to the planning page

Reply:
oh okay, yeah it makes sense